test/functional/http/get/transactions.js

```diff
@@ -455,14 +452,14 @@ describe('GET /api/transactions', function () {
 				params = [
 					'orderBy=' + sortField
 				];
-				return getTransactionsPromise(params).then(function (res) {
+				return getTransactionsPromise(params.join('&')).then(function (res) {
 					node.expect(res).to.have.property('success').to.be.ok;
 					node.expect(res).to.have.property('transactions').that.is.an('array');
 
 					var dividedIndices = res.transactions.reduce(function (memo, peer, index) {
 						memo[peer[sortField] === null ? 'nullIndices' : 'notNullIndices'].push(index);
 						return memo;
-					}, {notNullIndices: [], nullIndices: []});
+					}, { notNullIndices: [], nullIndices: [] });
```

Comment:
why spaces here?

Reply:
Eslint does not complain, my editor suggested me.